managed-apps-vision/README.md

```diff
@@ -0,0 +1,30 @@
+# Managed Apps Vision
+
+This RFC describes a general vision for improving the user experience of Managed Apps.
+
+With a previous iteration of the structure of Giant Swarm, one team was responsible for all of the managed apps. As the number of apps that Giant Swarm managed increased, it became more difficult to own these apps.
+
+Giant Swarm has updated its structure so that ownership of apps is spread across multiple teams - instead of one team owning all apps, each team owns a number of apps. Each team is now responsible for an area of solutions, such as monitoring, and are then responsible for apps that help in that area, such as Prometheus. This ownership also includes concepts such as trainings or sales enablement. We've been referring to this concept as 'holistic ownership'.
```

Comment:
```suggestion
Giant Swarm has updated its structure so that ownership of apps is spread across multiple teams - instead of one team owning all apps, each team owns a number of apps. Each team is now responsible for a solution to a specific need. This solution is typically made up of one ore more open source apps. An example of such a solution is monitoring. The team is responsible for apps that help in that area, such as Prometheus. This ownership also includes concepts such as trainings and sales enablement. We've been referring to this concept as 'holistic ownership'.
```

Reply:
not sure that's right, teams own a "field/area of solutions" for several interrelated needs, not just one solution for one need, and depending on use cases you might have several of those needs or just one of them for now, e.g. for obersvability the team might own a tracing solution but not everyone might have the use cases for that.

Reply:
i think it kind of depends on how you model the solutions - you can keep them at a general level of 'solve monitoring', or a more specific level of 'solve monitoring problem a', or 'solve monitoring problem b' etc.

i do think area of solutions makes more sense though, because teams don't have a _single_ solution right now